From f39bf80fc9af504b2700846870f7734d64cd1f36 Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Wed, 19 Jun 2019 16:05:02 +0800
Subject: [PATCH 1/2] drivers: pinctrl-imx: fix kernel warning unused variable
 'config'

Since regmap related code is removed by previous patch, need remove
related declaration to prevent compile warning:

drivers/pinctrl/freescale/pinctrl-imx.c: In function 'imx_pinctrl_probe':
drivers/pinctrl/freescale/pinctrl-imx.c:573:23: warning: unused variable 'config' [-Wunused-variable]
  struct regmap_config config = { .name = "gpr" };
                       ^~~~~~

Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 drivers/pinctrl/freescale/pinctrl-imx.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/pinctrl/freescale/pinctrl-imx.c b/drivers/pinctrl/freescale/pinctrl-imx.c
index 9617d956db7f..88f6341837db 100644
--- a/drivers/pinctrl/freescale/pinctrl-imx.c
+++ b/drivers/pinctrl/freescale/pinctrl-imx.c
@@ -570,7 +570,6 @@ static void imx_free_resources(struct imx_pinctrl *ipctl)
 int imx_pinctrl_probe(struct platform_device *pdev,
 		      const struct imx_pinctrl_soc_info *info)
 {
-	struct regmap_config config = { .name = "gpr" };
 	struct device_node *dev_np = pdev->dev.of_node;
 	struct pinctrl_desc *imx_pinctrl_desc;
 	struct device_node *np;
-- 
2.17.1

